datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/db.yaml

```diff
@@ -25,14 +25,33 @@
                  The number of columns must equal the number of columns returned in the query.
                  There are 2 required pieces of data:
                    a. name - The suffix to append to `<INTEGRATION>.` to form
-                             the full metric name. If `type` is `tag`, this column is
+                             the full metric name. If `type` is a `tag` type, this column is
                              considered a tag and applied to every
                              metric collected by this particular query.
                    b. type - The submission method (gauge, monotonic_count, etc.).
```

Comment:
To reviewer: My main motive for this part is to highlight the tag types `tag_list` and `tag_not_null` so users can use them in custom queries, although I wasn't sure if these full descriptions would make the conf.yaml.example too messy/long winded. Thoughts are appreciated on the best way to convey these types!